readthedocs/projects/version_handling.py

```diff
@@ -109,7 +113,8 @@ def version_windows(versions, major=1, minor=1, point=1):
 def parse_version_failsafe(version_string):
     try:
         return Version(
-            unicodedata.normalize('NFKD', unicode(version_string)).encode('ascii', 'ignore')
+            six.text_type(
+                unicodedata.normalize('NFKD', six.text_type(version_string)))
```

Comment:
I believe I removed the `encode('ascii', 'ignore')` because in Python 3 it was already a unicode string, so it was erroring out on that call. I don't believe the call to `six.text_type` is necessary, but I'm not 100% sure. Unicode handling in 2 is a PITA :/